qemu/tests/block_hotplug.py

```diff
@@ -67,12 +68,15 @@ def get_new_disk(disk1, disk2):
     get_disk_cmd = params.get("get_disk_cmd")
     context_msg = "Running sub test '%s' %s"
     device_list = []
+    disk_index = params.objects("disk_index")
+    disk_letter = params.objects("disk_letter")
```

Comment:
Instead of using the static values, I think we should have a function for this (e.g. `get_unused_disk_letter`).